app/controllers/admin/content_controller.rb

```diff
@@ -113,6 +113,20 @@ def autosave
     render :text => nil
   end
 
+  def merge
+    @article = Article.find(params["id"])
+    merge_article = Article.find(params[:merge_with])
+    @article.body += merge_article.body
+    @article.save
+    merge_article.comments.each do |comment|
+      comment_copy = comment.clone
+      comment_copy.article_id = @article.id
+      comment_copy.save
+    end
+    merge_article.destroy
```

Comment:
Since the `@article` update and the `merge_article` deletion depend on one another, it would be good to use a transaction to ensure that both or none would happen.